Description: Fix failing tests with pytest 3.3
 Reverse an upstream patch to make tests run successfully with pytest 3.3
Author: Quentin Long <qlo@yelp.com>
Origin: https://github.com/Yelp/elastalert/commit/e0cab84634dc8cb38e91a954b340f60a8158c059
Last-Update: 2018-03-30

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -2,7 +2,7 @@ coverage
 flake8
 pre-commit
 pylint<1.4
-pytest<3.3.0
+pytest
 setuptools
 sphinx_rtd_theme
 tox<2.0
--- a/tests/base_test.py
+++ b/tests/base_test.py
@@ -1048,7 +1048,7 @@ def test_wait_until_responsive(ea):
     ]
 
 
-def test_wait_until_responsive_timeout_es_not_available(ea, capsys):
+def test_wait_until_responsive_timeout_es_not_available(ea, caplog):
     """Bail out if ElasticSearch doesn't (quickly) become responsive."""
 
     # Never becomes responsive :-)
@@ -1064,8 +1064,7 @@ def test_wait_until_responsive_timeout_e
         assert exc.value.code == 1
 
     # Ensure we get useful diagnostics.
-    output, errors = capsys.readouterr()
-    assert 'Could not reach ElasticSearch at "es:14900".' in errors
+    assert 'Could not reach ElasticSearch at "es:14900".' in caplog.text
 
     # Slept until we passed the deadline.
     sleep.mock_calls == [
@@ -1075,7 +1074,7 @@ def test_wait_until_responsive_timeout_e
     ]
 
 
-def test_wait_until_responsive_timeout_index_does_not_exist(ea, capsys):
+def test_wait_until_responsive_timeout_index_does_not_exist(ea, caplog):
     """Bail out if ElasticSearch doesn't (quickly) become responsive."""
 
     # Never becomes responsive :-)
@@ -1091,8 +1090,7 @@ def test_wait_until_responsive_timeout_i
         assert exc.value.code == 1
 
     # Ensure we get useful diagnostics.
-    output, errors = capsys.readouterr()
-    assert 'Writeback index "wb" does not exist, did you run `elastalert-create-index`?' in errors
+    assert 'Writeback index "wb" does not exist, did you run `elastalert-create-index`?' in caplog.text
 
     # Slept until we passed the deadline.
     sleep.mock_calls == [
